scripts/zones/Ceizak_Battlegrounds/mobs/Mastop.lua

```diff
@@ -0,0 +1,73 @@
+-----------------------------------
+-- Area: Ceizak Battlegrounds
+-- NM: Mastop
+-- !pos -240 0 36 261
+-- !additem 6014
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+
+local boomingMobSkillId = 2770    -- ID for the Booming Bombination skill
+local reductionIncrement = 2500   -- Increment value for damage reduction per use
+local damageCap = -10000          -- Maximum physical damage resistance (-100%)
+local initialDamage = 0           -- Initial physical damage resistance (0%)
+local auraDuration = 60           -- Duration of the aura effects in seconds
+local auraChance = 20             -- 10% chance to apply an aura effect
+local boomingCD = 10              -- Cooldown period in seconds for Booming Bombination
+
+local function handleBoomingBombination(mob, target)
+    local boomingUsageCount = mob:getLocalVar('boomingUsageCount') or 0
+    local reduction = initialDamage - (boomingUsageCount * reductionIncrement)
```

Comment:
Parenthesis are unnecesary here. Also, if you can align the `=` it will make reading it much more easy.